CMakeLists.txt

```diff
@@ -44,11 +44,25 @@ else()
     set(SFML_BUILD_EXAMPLES FALSE)
 endif()
 
+# add options to select which modules to build
+sfml_set_option(SFML_BUILD_SYSTEM TRUE BOOL "TRUE to build SFML's System module. This setting is ignored, if any of the other modules is built.")
```

Comment:
Indeed, I forget the issue at hand, sorry.

But wouldn't someone who's building the doc want to build at least one module of SFML?